ibu-imager/cmd/create.go

```diff
@@ -79,9 +80,14 @@ func create() {
 
 	// Check if containerRegistry was provided by the user
 	if containerRegistry == "" {
```

Comment:
I believe we can just use https://pkg.go.dev/github.com/spf13/cobra#MarkFlagRequired